api/RTCIceTransport.json

```diff
@@ -407,6 +511,58 @@
           }
         }
       },
+      "statechange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport/statechange_event",
+          "description": "<code>statechange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
This [landed in 70](https://storage.googleapis.com/chromium-find-releases-static/35a.html#35aa0ef49c836db358b8c935a58e1cf4f726441e).